lib/language_pack/ruby.rb

```diff
@@ -1099,7 +1099,7 @@ def node_preinstall_bin_path
     return @node_preinstall_bin_path if defined?(@node_preinstall_bin_path)
 
     legacy_path = "#{Dir.pwd}/#{NODE_BP_PATH}"
-    path        = run("which node")
+    path        = run("which node").chomp
```

Comment:
Do you need a `chomp` on the yarn version of this line below?

Reply:
Great call, I added it.